.travis.yml

```diff
@@ -1,10 +1,25 @@
 language: python
-python:
-  - "2.6"
-  - "2.7"
-install:
-  - "pip install -e .[testing] --use-mirrors"
-script: nosetests
 branches:
-  only:
-      - develop
+    only:
+        - develop
+        - 2to3
+
+before_install:
+    # In python 2.7 we have scipy/numpy through the Ubuntu package manager
```

Comment:
If this works, great! Another option is to install scipy/cython/numpy with conda, which will be much faster. For an example, see here:
https://github.com/xray/xray/blob/master/.travis.yml